eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
do we need to import print_function from futures here?

Reply:
print is not used anymore on the last commit (a08e8c4)